### PR TITLE
feat: (farms) Use Text component instead of Subtle and Title

### PR DESCRIPTION
**Pull request by memoyil**
To review:

https://deploy-preview-1492--pancakeswap-dev.netlify.app/

There is a Todo above Subtle style component decleration which is waiting to be fixed.

https://github.com/pancakeswap/pancake-frontend/blob/5bc14994d8cd1334adb48acf0c40a2e68162c64b/src/views/Farms/components/FarmTable/Actions/styles.ts#L36

### CODE REVIEW

File: src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx [hunk 1]
```diff
@@ -53,8 +53,12 @@ const HarvestAction: React.FunctionComponent<HarvestActionProps> = ({ pid, userD
   return (
     <ActionContainer>
       <ActionTitles>
-        <Title>CAKE </Title>
-        <Subtle>{t('Earned').toUpperCase()}</Subtle>
+        <Text bold textTransform="uppercase" color="secondary" fontSize="12px" pr="3px">
```
Comment (hachiojidev) [inline src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx:56]: Spacing need to be in units of 4px.

File: src/views/Farms/components/FarmTable/Actions/StakedAction.tsx [hunk 2]
```diff
@@ -154,8 +160,12 @@ const Staked: React.FunctionComponent<StackedActionProps> = ({
     return (
       <ActionContainer>
         <ActionTitles>
-          <Subtle>{t('Stake').toUpperCase()} </Subtle>
-          <Title>{lpSymbol}</Title>
+          <Text bold textTransform="uppercase" color="textSubtle" fontSize="12px" pr="3px">
```
Comment (hachiojidev) [inline src/views/Farms/components/FarmTable/Actions/StakedAction.tsx:163]: Same here.

**Review by hachiojidev — CHANGES_REQUESTED** [review]

File: src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx (hunk 1 above)
Comment (memoyil) [reply src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx]: Done

File: src/views/Farms/components/FarmTable/Actions/StakedAction.tsx (hunk 2 above)
Comment (memoyil) [reply src/views/Farms/components/FarmTable/Actions/StakedAction.tsx]: Done

File: src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx (hunk 1 above)
Comment (memoyil) [reply src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx]: I changed spacing in farm cards to 4px too

**Comment by monstro805** [comment]
@mando8051
Sent from Yahoo Mail on Android 
 
  On Sun, Jun 13, 2021 at 4:30 PM, ***@***.***> wrote:   


@memoyil commented on this pull request.

In src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx:
> @@ -53,8 +53,12 @@ const HarvestAction: React.FunctionComponent<HarvestActionProps> = ({ pid, userD
   return (
     <ActionContainer>
       <ActionTitles>
-        <Title>CAKE </Title>
-        <Subtle>{t('Earned').toUpperCase()}</Subtle>
+        <Text bold textTransform="uppercase" color="secondary" fontSize="12px" pr="3px">

I changed spacing in farm cards to 4px too

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**